src/signals/incident-management/containers/FilterTagList/__tests__/FilterTagList.test.js

```diff
@@ -130,16 +128,39 @@ describe('signals/incident-management/containers/FilterTagList', () => {
         )
       );
 
-      expect(
-        queryByText(`${maincategory_slug[0].value}${allLabelAppend}`)
-      ).toBeTruthy();
+      expect(queryByText(`${maincategory_slug[0].value}${allLabelAppend}`)).toBeTruthy();
     });
 
     it('renders a list of tags', () => {
       const { queryAllByTestId, queryByText } = render(
-        withAppContext(
+        withContext(
+          <FilterTagListComponent tags={tags} subCategories={subCategories} mainCategories={mainCategories} />
+        )
+      );
+
+      expect(queryByText('Normaal')).toBeInTheDocument();
+      expect(queryByText('Tevreden')).toBeInTheDocument();
+      expect(queryByText('februariplein 1')).toBeInTheDocument();
+      expect(queryByText('Centrum')).toBeInTheDocument();
+      expect(queryByText('Westpoort')).toBeInTheDocument();
+      expect(queryByText('North')).not.toBeInTheDocument();
+      expect(queryByText('Telefoon – Adoptant')).toBeInTheDocument();
+      expect(queryByText('Telefoon – ASC')).toBeInTheDocument();
```

Comment:
I know it has been violated in other places in the code base as well, but it's better not to use hardcoded strings, but to reference props from the appropriate fixtures. If we replace the fixture, this test can (and most likely will) fail.